client/src/pages/profile/index.tsx

```diff
@@ -80,375 +43,185 @@ const Profile = (): ReactElement => {
       }
     };
 
-    const loadCurrentUserPermissions = async (): Promise<void> => {
-      const res = await getUserPermissionsByID(userId);
+    const getPitches = async (): Promise<void> => {
+      const res = await getAggregatedUser(userId);
 
       if (!isError(res)) {
-        setPermissions(res.data.result);
+        // pitches the user created
+        const submittedPitches = res.data.result.aggregated.submittedPitches;
+
+        // pitches the user claimed a team for
+        const claimedPitches = res.data.result.aggregated.claimedPitches;
+
+        // filter assignment contributors to get teams claimed by user for pitch
+        claimedPitches.forEach((pitch) => {
+          pitch.assignmentContributors = pitch.assignmentContributors?.filter(
+            (contributor) => contributor.userId === userId,
+          );
+        });
```

Comment:
First, why is this happening?
Second, why not just set every pitch.assignmentContributor array in the claimed picthes to be equal to `[userId]` if this is necessary. I am not sure why the claimed pitch assignment contributors need to change though.

Reply:
This was for the column in the contributions table `teams you're on`. I thought I could get this from assignment contributors because it stores the teams the user had been assigned to for the pitch. Since the table uses records of type pitches in the dynamic table I thought of passing this data through the pitches assignment contributors.